pkg/loader/compose/compose.go

```diff
@@ -358,6 +358,11 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		if len(serviceConfig.Port) == 0 && composeServiceConfig.Labels["kompose.service.type"] == "NodePort" || composeServiceConfig.Labels["kompose.service.type"] == "LoadBalancer" {
+			log.Fatal("Failed to proceed as Nodeport or loadbalancer is defined without ports")
```

Comment:
Was just thinking again (sorry for asking for another change) that this could be better:
```go
log.Fatal("%s defined with no ports present. Issues may occur when bringing up artifacts.", composeServiceConfig.Labels["kompose.service.type"])
```

And also change this log to warning instead? What do you think @kadel ?